pkg/pool-linear/test/LinearPoolFactory.test.ts

```diff
@@ -22,19 +22,17 @@ describe('LinearPoolFactory', function () {
   const POOL_SWAP_FEE_PERCENTAGE = fp(0.01);
   const BASE_PAUSE_WINDOW_DURATION = MONTH * 3;
   const BASE_BUFFER_PERIOD_DURATION = MONTH;
-  const WRAPPED_TOKEN_RATE_CACHE_DURATION = MONTH;
 
   before('setup signers', async () => {
     [, owner] = await ethers.getSigners();
   });
 
   sharedBeforeEach('deploy factory & tokens', async () => {
     vault = await Vault.create();
-    factory = await deploy('LinearPoolFactory', { args: [vault.address] });
+    factory = await deploy('MockLinearPoolFactory', { args: [vault.address] });
```

Comment:
This is fine as a temporary measure to have these tests pass, but in the future we'll want to replace this back with the AaveLinearPoolFactory.